docs/multiple_updates_for_same_buffer.md

```diff
@@ -0,0 +1,76 @@
+This is copied from #197
+
+### Multiple updates for a SSBO in VkRunner
+
+As debated [here](https://github.com/Igalia/vkrunner/issues/46),
+VkRunner applies multiple updates for a SSBO and uses the final
+one for all computes or draws.
+
+For example,
+```
+ssbo 0 subdata float 0 1.0
+compute 1 1 1
+ssbo 0 subdata float 0 2.0
+compute 1 1 1
+ssbo 0 subdata float 36 1.0 2.0 3.0 \
+                        ... ( more than 4000 numbers) ... 4096.0
+compute 1 1 1
+probe ssbo float 0 ~= 2.0
+ssbo 0 subdata float 4000 1.0 2.0 3.0 \
+                          ... ( more than 4000 numbers) ... 4096.0
+compute 1 1 1
+probe ssbo float 4000 ~= 1.0
+```
+
+This script has the same behavior as the following script does
+
+```
+ssbo 0 subdata float 0 1.0
+ssbo 0 subdata float 0 2.0
+ssbo 0 subdata float 36 1.0 2.0 3.0 \
+                        ... ( more than 4000 numbers) ... 4096.0
+ssbo 0 subdata float 4000 1.0 2.0 3.0 \
+                          ... ( more than 4000 numbers) ... 4096.0
+
+compute 1 1 1
+compute 1 1 1
+compute 1 1 1
+probe ssbo float 0 ~= 2.0
+compute 1 1 1
+probe ssbo float 4000 ~= 1.0
+```
+
+### Multiple updates for a SSBO in Amber script
+
+Amber script will have different semantics for those multiple
```

Comment:
If Amber is running a vkscript then it should behave the same as VkRunner.

Reply:
I don't think we want follow what VkRunner does here as the script is confusing. I think we should deviate from VkRunner in this case. We just have to document that we have deviated.